TemporaryOysters/bangazon_website
# Me order testing

##Pull Requests



##Descriptions:

* Please describe what issue this PR solves
This PR tests and creates "Bangazon Orders"










##Steps to Test:


To test this code, please 
1. `git fetch`
2. Checkout the`me-OrderTesting` branch
3. Delete only migrations inside `bangazon_webstore/migrations/` (example `0001_initial.py`), no other files
4. Delete your own `db.sqlite3` file
5. From terminal, within your main `bangazon_website` folder, run `python bangazon_website/manage.py makemigrations bangazon_webstore`
6. From same folder run `python bangazon_website/manage.py migrate`
7. From the same folder run `python bangazon_website/manage.py test bangazon_webstore`
8. You should have 16 tests passing

##Related Issue:


#15 
##System configuration.


* Are there 3rd party libraries that need to be installed?
None

* Any command line utilities to run?
See above
